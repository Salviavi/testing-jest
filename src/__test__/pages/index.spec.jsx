import HomeView from "@/pages";
import { render } from "@testing-library/react";

describe("Home Page", () => {
    it("should render home page", () => {
        const {container} = render(
        <HomeView></HomeView>
    )
        expect(container).toMatchSnapshot();
    })
})