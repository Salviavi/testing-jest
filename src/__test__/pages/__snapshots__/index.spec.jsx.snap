// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render home page 1`] = `
<div>
  <div>
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
